src/Mailer.js

```diff
@@ -134,10 +134,14 @@ export default class Mailer extends templater.Templater {
 
             if (this.config.smtp_relay.pretend) {
 
-                (this._logger.debug || this._logger.info)(`pretending to send email to ${this._to}`, this._html);
+                const pretend_to_emails = this._to || [];
+                const pretend_bcc_emails = this._bcc || [];
+                const pretend_emails = pretend_bcc_emails.concat(pretend_to_emails);
+
+                (this._logger.debug || this._logger.info)(`pretending to send email to ${pretend_emails}`, this._html);
```

Comment:
Well, I don't think there's an requirement for the accepted format, so why don't we build an object and just print it?
```
const pretend_emails = {
 to: this._to,
 cc: this._cc,
 bcc: this._bcc
}
(this._logger.debug || this._logger.info)('pretending to send email to', pretend_emails, this._html);
```

Reply:
resolved in 
46e943d